specs/altair/beacon-chain.md

```diff
@@ -99,6 +98,8 @@ Altair is the first beacon chain hard fork. Its main features are:
 | Name | Value |
 | - | - |
 | `G2_POINT_AT_INFINITY` | `BLSSignature(b'\xc0' + b'\x00' * 95)` |
+| `PARTICIPATION_FLAGS` | `3` |
+| `PARTICIPATION_FLAG_WEIGHTS` | `[TIMELY_HEAD_WEIGHT, TIMELY_SOURCE_WEIGHT, TIMELY_TARGET_WEIGHT]` |
```

Comment:
i can't comment up above but in this case, we should get rid of the constants `TIMELY_*_INDEX` defined up above